honeybee_energy/load/lighting.py

```diff
@@ -102,9 +105,12 @@ def radiant_fraction(self):
 
     @radiant_fraction.setter
     def radiant_fraction(self, value):
-        self._radiant_fraction = float_in_range(
-            value, 0.0, 1.0, 'lighting radiant fraction')
-        self._check_fractions()
+        if value is not None:
+            self._radiant_fraction = float_in_range(
+                value, 0.0, 1.0, 'lighting radiant fraction')
+            self._check_fractions()
+        else:
+            self._radiant_fraction = 0.32
```

Comment:
This can end up creating strange behavior since you are not checking the fraction afterwards. Is there a reason that we hard-core the value here and also a reason for not calling `check_fractions`?

The same question about other fields with none 0 values.

Reply:
Very good catch.  I forgot that this is only appropriate when the default is 0.